### PR TITLE
47 delete uneeded code

### PR DESCRIPTION
**Pull request by ErinGreenhalgh**
Closes #52 
Deletes unneeded tests and corresponding code:
-deletes all tests pertaining to cart, cart items, orders, and order items
-deletes all routes, controllers, models, and view references pertaining to cart, orders, cart items, and order items
-creates migrations to drop orders and order_items tables and modifies seed data to remove orders

-The only cart references left are in the custom.scss file. We can decide if we want to keep this css and replace "cart" with "bid" or delete it and write new css for bids when we come to styling

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -9,11 +9,7 @@ def create
     @user = User.find_by(username: params[:session][:username])
     if @user && @user.authenticate(params[:session][:password])
       session[:user_id] = @user.id
-      if session[:login_redirect] == "/cart"
```
Comment (ErinGreenhalgh) [inline app/controllers/sessions_controller.rb:12]: we'll need to recreate this functionality for redirecting back to the item page when they log in from an item's show page

File: db/migrate/20160701002236_drop_order_items_table.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class DropOrderItemsTable < ActiveRecord::Migration
+  def change
+    drop_table :order_items
```
Comment (seeker105) [inline db/migrate/20160701002236_drop_order_items_table.rb:3]: Good use of migrations, to change schema!

**Comment by seeker105** [comment]
Looks like a great job of filtering out unneeded code.

File: spec/support/factories.rb [hunk 3]
```diff
@@ -48,7 +48,7 @@
     price "5.99"
     image "http://i.imgur.com/kgOqHMk.gif"
     status "open"
-    end_time Time.now 
+    end_time Time.now
 
     factory :item_with_bids do
       transient do
```
Comment (seeker105) [inline spec/support/factories.rb:54]: Great change - increases the flexibility of the factory!

**Pull request merged**